ui/pages/send/send-content/send-content.component.js

```diff
@@ -84,24 +92,48 @@ export default class SendContent extends Component {
     );
   }
 
+  checkContractAddress = () => {
```

Comment:
Can we make this method async so we can use `async/await` instead of `then`?

Reply:
Yes, we can. I changed it.
Fixed in: 51054012cfbb3e51b2853adafa9e9d8fd155656d